migrations/mystery_items.js

```diff
@@ -27,6 +27,9 @@ if (_id) {
   db.users.update({_id:_id}, update);
 } else {
   // multiple (once @ start of event)
-  db.users.update({'purchased.plan.customerId':{$ne:null}}, update, {multi:true});
-}
-
+  db.users.update([{$and: [{'purchased.plan.customerId':{$ne:null}},
```

Comment:
If @alys and @paglias can sign off on the query, the only thing I'd change is formatting:

```js
db.users.update([
  { $and: [
    { 'purchased.plan.customerId': { $ne: null } },
    { $or: [
      { 'purchased.plan.dateTerminated': { $gte: new Date() }},
      { 'purchased.plan.dateTerminated': { $exists: false } },
      { 'purchased.plan.dateTerminated': { $eq: null } }
    ]}
  ]}
], update, { multi: true });
```